AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -924,6 +924,7 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
                                             * release notes.
                                             */
                                            self.releaseDetails = nil;
+                                           self.updateFlowInProgress = NO;
```

Comment:
Interesting, Android has another way of resolving that issue. The workflow is still in progress but we remember the sub-state. When app resumes we restore the dialog that corresponds to the current step. It is unclear to me here how it will work on iOS. Will we restart from scratch with making the API call again? That leaves some time where app can be used and there is a mandatory update (that should block ui).

Reply:
It clears all states/data at this time and starts over the process when an app is back to foreground from browser. Even I didn't realize it is working like that but it is a separate topic for the PR.